### PR TITLE
Added clarification for the window.pageYOffset property

### PR DESCRIPTION
**Pull request by ghost**
This is for the issue #18735
This sentence should add some clarification to the ``window.pageYOffset property`` once it is set. 

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/window/scrolly/index.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ property:
 ```js
 window.pageYOffset === window.scrollY; // always true
 ```
+Setting `pageYOffset` property creates a property with the same name on the platform object. 
+This new property will influences the intial property.
```
Comment (Josh-Cena) [inline files/en-us/web/api/window/scrolly/index.md:66]: It's not really clear how this addresses https://github.com/mdn/content/issues/18735. That one is pointing out that `pageYOffset` and `scrollY` are independent models and setting one does not reactively update the other.

image

Both are accessors on `window`, so you don't really "create a property" anywhere.

Comment (najmiehsa) [reply files/en-us/web/api/window/scrolly/index.md:66]: Thank you for the review. I update this.

Comment (najmiehsa) [reply files/en-us/web/api/window/scrolly/index.md:66]: I close this one because it take some time for me to do a well-rounded research for the issue.

**Comment by najmiehsa** [comment]
Closing this in favor of  this PR https://github.com/mdn/content/pull/19690

**Pull request closed without merging**